Library/Homebrew/brew.rb

```diff
@@ -80,7 +80,13 @@
   # - no arguments are passed
   if empty_argv || help_flag
```

Comment:
Does this work if you add `cmd != "cask"` here and falls into the code below as expected? If so that seems like a more minimal and cleaner solution. If you change to that and it works: please add the `# Let Cask handle the help command` comment, too, thanks!

Reply:
Yes, it works! I've simplified it below. Thanks for the comment! 👍